tark/tark_web/templates/mane_data_access.html

```diff
@@ -24,7 +24,12 @@ <h1 class="">Accessing MANE data</h1>
                     <span class="glyphicon glyphicon-info-sign"></span>
                     Note: We will have several data releases as we make progress on completing the MANE projects.
                 </div>
+                <!-- Browse list on access mane data to be above Bulk data download -->
```

Comment:
This comment can be removed as we can see in the code that it's above Bulk data

Reply:
@bilalebi , Thank you Bilal..This is now done.